docs/shared/react-tutorial/react-monorepo.md

```diff
@@ -0,0 +1,1039 @@
+    },
+    {
+      "name": "inventory-e2e",
+      "type": "e2e",
+      "data": {
+        "tags": []
+      }
+    },
+    {
+      "name": "shared-ui",
+      "type": "lib",
+      "data": {
+        "tags": []
+      }
+    },
+    {
+      "name": "orders",
+      "type": "lib",
+      "data": {
+        "tags": []
+      }
+    },
+
+    {
+      "name": "products",
+      "type": "lib",
+      "data": {
+        "tags": []
+      }
+    }
+  ],
+  "dependencies": {
+    "react-store": [
+      { "source": "react-store", "target": "orders", "type": "static" },
+      { "source": "react-store", "target": "products", "type": "static" }
+    ],
+    "react-store-e2e": [
+      {
+        "source": "react-store-e2e",
+        "target": "react-store",
+        "type": "implicit"
+      }
+    ],
+    "inventory": [
+      { "source": "inventory", "target": "products", "type": "static" }
+    ],
+    "inventory-e2e": [
+      { "source": "inventory-e2e", "target": "inventory", "type": "implicit" }
+    ],
+    "shared-ui": [],
+    "orders": [],
+    "products": []
+  },
+  "workspaceLayout": { "appsDir": "", "libsDir": "" },
+  "affectedProjectIds": [
+    "products",
+    "inventory",
+    "inventory-e2e",
+    "react-store",
+    "react-store-e2e"
+  ],
+  "focus": null,
+  "groupByFolder": false
+}
+```
+
+{% /graph %}
+
+## Building the Apps for Deployment
+
+<!-- {% video-link link="https://youtu.be/OQ-Zc5tcxJE?t=856" /%} -->
+
+If you're ready and want to ship your applications, you can build them using
+
+```{% command="npx nx run-many -t build" path="react-monorepo" %}
+// todo
+vite v4.3.5 building for production...
+✓ 33 libs transformed.
+dist/react-store/index.html                   0.48 kB │ gzip:  0.30 kB
+dist/react-store/assets/index-e3b0c442.css    0.00 kB │ gzip:  0.02 kB
+dist/react-store/assets/index-378e8124.js   165.64 kB │ gzip: 51.63 kB
+✓ built in 496ms
+
+ ——————————————————————————————————————————————————————————————————————————————————————————————————————————
+
+ >  NX   Successfully ran target build for project reactutorial (1s)
+```
+
+All the required files will be placed in `dist/react-store` and `dist/inventory` and can be deployed to your favorite hosting provider.
+
+You can even create your own `deploy` task that sends the build output to your hosting provider.
+
+```json {% fileName="apps/react-store/project.json" %}
+{
+  "targets": {
+    "deploy": {
+      "dependsOn": "build",
+      "command": "netlify deploy --dir=dist/react-store"
+    }
+  }
+}
+```
+
+Replace the `command` with whatever terminal command you use to deploy your site.
+
+The `"dependsOn": "build"` setting tells Nx to make sure that the project's `build` task has been run successfully before the `deploy` task.
+
+With the `deploy` tasks defined, you can deploy a single application with `nx deploy react-store` or deploy any applications affected by the current changes with:
+
+```shell
+nx affected -t deploy
+```
+
+### Caching
```

Comment:
I think this accidentally finished up in the wrong section. Should have been in the "Testing and Linting ..." section.